crates/dpapi-core/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![warn(clippy::std_instead_of_alloc)]
+#![warn(clippy::std_instead_of_core)]
+// #![warn(missing_docs)]
+
+extern crate alloc;
+
+mod decode;
+mod encode;
+mod error;
+pub mod gkdi;
+mod marker;
+mod padding;
+pub mod rpc;
+pub mod str;
+
+pub use decode::{DecodeWithContextOwned, FindLength, FixedPartSize, decode_uuid, read_c_str_utf16_le};
+pub use encode::{EncodeVec, encode_seq, encode_uuid, size_seq};
+pub use error::{Error, Result};
+pub use ironrdp_core as core;
```

Comment:
suggestion: Instead of re-exporting ironrdp_core entirely, re-export the traits and other types you actually need only. E.g.: `Encode`/`Decode` traits.

Reply:
done